website/src/components/ReviewPage/ReviewPage.tsx

```diff
@@ -149,50 +166,60 @@ const InnerReviewPage: FC<ReviewPageProps> = ({ clientConfig, organism, group, a
-        },
-        {
-            text: 'sequences with errors',
-            countNumber: errorCount,
-            setVisibility: setShowErrors,
-            visibilityEnabled: showErrors,
-        },
-    ];
-
     const sequences: SequenceEntryStatus[] = sequencesData.sequenceEntries;
 
     const controlPanel = (
         <div className='flex flex-col'>
             <div className='text-gray-600 mr-3'>
-                {unfinishedCount > 0 && (
+                {unprocessedCount > 0 && (
                     <span className='loading loading-spinner loading-sm mr-2 relative top-1'> </span>
                 )}
-                {finishedCount} of {total} sequences processed
+                {processedCount} of {total} sequences processed
             </div>
-            <div className='border border-gray-200 rounded-md p-3 mt-3 flex gap-3'>
-                <LucideFilter className='w-4 h-4 mr-1.5 inline-block text-gray-500 mt-0.5' />
-                {categoryInfo.map((info, index) => {
-                    return <NumberAndVisibility key={index} {...info} />;
-                })}
+            <div className='border border-slate-200 p-3 mt-3 flex items-start'>
+                <LucideFilter className='w-4 h-4 inline-block text-gray-500 mt-1 mr-3' />
+                <NumberAndVisibility
+                    key='unprocessed'
+                    text='awaiting processing'
+                    countNumber={unprocessedCount}
+                    setVisibility={setShowUnprocessed}
+                    visibilityEnabled={showUnprocessed}
+                />
+                <div className='border-green-500 border-b-2 pb-1'>
+                    <NumberAndVisibility
+                        key='valid'
+                        text='no issues'
+                        countNumber={noIssuesCount}
+                        setVisibility={setShowNoIssues}
+                        visibilityEnabled={showNoIssues}
+                    />
+                </div>
+                <div className='border-yellow-400 border-b-2 pb-1'>
+                    <NumberAndVisibility
+                        key='warnings'
+                        text='with warnings'
+                        countNumber={warningCount}
+                        setVisibility={setShowWarnings}
+                        visibilityEnabled={showWarnings}
+                    />
+                </div>
+                <div className='border-b-2 border-red-600 pb-1'>
+                    <NumberAndVisibility
+                        key='errors'
+                        text='with errors'
+                        countNumber={errorCount}
+                        setVisibility={setShowErrors}
+                        visibilityEnabled={showErrors}
+                    />
+                </div>
             </div>
         </div>
     );
 
     const pagination = (
         <div className='flex justify-end align-center gap-3 py-3'>
             <Pagination
```

Comment:
Something very small: if you first go to the last page and then filter out sequences, you end up on an empty page. Instead, I think that it would be better to go e.g. to the last of the existing pages:

https://github.com/user-attachments/assets/5d0d1f29-5de8-4860-aaf0-133dbfd4e110

But definitely feel free to move this to a new issue!

Reply:
Should be fixed here: [979b7c0](https://github.com/loculus-project/loculus/pull/3125/commits/979b7c0d772a02ccd3d846c42dac819da0356ddd)

Reply:
I didn't think about testing this, very good!!